lab/hub.go

```diff
@@ -49,6 +49,7 @@ func NewHub(ctx context.Context, creator Creator, buffer int, cap int) (*hub, er
 		ctx := context.Background()
 		for range ready {
 			wg.Add(1)
+			// todo: handle this in case of error
```

Comment:
this is a critical error  that has to be solved. it will cause haaukins crash in some cases

Reply:
however, it is NOT topic of this pull request or issue, hence, do not consider it for this pull request.